src/ppscore/calculation.py

```diff
@@ -279,16 +283,21 @@ def score(df, x, y, task=None, sample=5000, cv=None, **kwargs):
     """
 
     if cv is None:
-        # Did not pass any CV - fallback to defaults
+
+        # Did not pass any CV - fallback to defaults:
+        # Crossvalidation is stratifiedKFold for classification, KFold for regression
         cv = CV_ITERATIONS
+        df = df.sample(frac=1, random_state=RANDOM_SEED, replace=False)
```

Comment:
I have a notion that this might be too easy. For example, when the user just wants to use 6 folds and passes cv=6, then we still need to perform the random resampling/shuffling in case of a normal KFold or stratifiedKFold.
Just in case of a CV strategy that needs to keep the order of the rows in the dataset, there should be no reshuffling.
What do you think about this?

Reply:
I thought you wanted to keep it in the default case only, hence I added it back as per your comment above. Alternatively, one can pass KFold with shuffle either True, or False, thus explicitly setting a CV procedure.

Reply:
In fact, setting CV to a train-test index generator as opposed to an integer means the user is doing it willingly, with  understanding on consequences and a specific purpose in mind

Reply:
Or maybe I just do not understand what you d like me to do here :)

Reply:
I think the most flexible and robust solution is to perform the default shuffling when the user passes no CV or an integer (assuming he wants to adjust the number of folds). In all the other cases, we do not have to shuffle as he has to pass a valid CV iterator.

What do you think about this?

Reply:
Ok i see. As of now it shuffles only if no CV is passed, and you're saying "let's shuffle the data if no CV is passed or if an integer is passed", is that correct ?

If the above is True, then I see your point and will adjust the if-statement above in the code accordingly.

Reply:
That looks fine, thank you :)